nifi-api/src/main/java/org/apache/nifi/controller/status/NodeStatus.java

```diff
@@ -35,7 +35,6 @@ public class NodeStatus implements Cloneable {
     private double processorLoadAverage;
 
     private long totalThreads;
-    private long eventDrivenThreads;
```

Comment:
This change is a helpful cleanup, which would be useful to handle separately. As this particular pull request may take some time to review, would you consider separating out this removal to its own pull request?